src/a-runtime-console/kubernetes/service/devnamespace.scope.ts

```diff
@@ -16,32 +17,32 @@ import { INamespaceScope } from './namespace.scope';
 export class DevNamespaceScope implements INamespaceScope {
   public namespace: Observable<string>;
 
-  private headers = new Headers({'Content-Type': 'application/json'});
+  private headers = new HttpHeaders({'Content-Type': 'application/json'});
```

Comment:
Isn't that something to be dealt by the interceptors?

Reply:
It should be unnecessary, since as a default the Interceptor should apply this same header. The Interceptors should also avoid overwriting any existing header with the same normalized name/key though, so it should be harmless for a service to specify headers in any situation - even if those headers would be redundant.

Reply:
Currently we can't use the present auth interceptor that's there in ngx-login-client because of issues with Mom release of v1 branch. So, the plan is to get through the angular upgrade and merge with latest branch of ngx-login-client using interceptor. Untill then we are explicitly adding auth headers in services.